script.service.janitor/resources/skins/Default/720p/DialogLogViewer.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
it is a good practice to clearly name your skin dialogs. Example: `script-service-janitor-DialogLogViewer.xml`. Same applies to other windows such as `skin.xml` you are also shipping